R/model.R

```diff
@@ -1197,6 +1202,7 @@ variational <- function(data = NULL,
 
   cmdstan_procs <- CmdStanProcs$new(
     num_procs = 1,
+    iter = ifelse(is.null(variational_args$iter),1e3,variational_args$iter),
```

Comment:
```suggestion
    iter = variational_args$iter %||% 1e3
```

Reply:
Also is 1e3 the right value for variational?

Reply:
good catch, should be 1e4